core/src/main/kotlin/fr/sncf/osrd/standalone_sim/ScheduleMetadataExtractorV2.kt

```diff
@@ -105,7 +105,7 @@ fun runScheduleMetadataExtractor(
                 rawInfra.getPhysicalSignalName(
                     loadedSignalInfra.getPhysicalSignal(pathSignal.signal)
                 )!!,
-                envelopeWithStops.interpolateTotalTime(sightOffset.distance.meters).seconds,
+                envelopeWithStops.interpolateDepartureFrom(sightOffset.distance.meters).seconds,
```

Comment:
```suggestion
                envelopeWithStops.interpolateArrivalAt(sightOffset.distance.meters).seconds,
```

Reply:
Done in https://github.com/OpenRailAssociation/osrd/pull/7609/commits/b7f044ebd08664f02948f7aafb19982fee469f65